### PR TITLE
fix: remove unnecessary configuration from tsconfig

### LINKED ISSUE #129: [Task]: Fix typescript config file
- [ ] Handle alias in svelte.config.js. [Reference](https://kit.svelte.dev/docs/configuration#alias)         
- [ ] Remove `baseUrl`, `paths` and `include/exclude` from tsconfig.json. This is handled by sveltekit.

### PR DESCRIPTION
**Pull request by VmMad**
Fixes #129

### CODE REVIEW

File: svelte.config.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ const config = {
 	preprocess: preprocess(),
 
 	kit: {
-		adapter: adapter()
+		adapter: adapter(),
+		alias: {
```
Comment (evavirseda) [inline svelte.config.js:12]: `alias` are already included in `vite.config.js` :mag:

**Review by evavirseda — APPROVED** [review]
LGTM :star:

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
devmode is broken in this PR

**Review by begonaalvarezd — APPROVED** [review]
mea culpa

**Pull request merged**